src/headless.esm.js

```diff
@@ -0,0 +1,60 @@
splayName=e.userDisplayName),t),yE=(e,t,r,a)=>(Kf(e,"userDisplayName",t.configuration.analytics.userDisplayName,it().analytics.userDisplayName,r,a),NC(e,t)),QC=e=>T({},t=>{t.addCase(mo,(r,a)=>{let n=Ls(r.configuration.accessToken);return n?VC(n,r,a.payload,e):r}).addCase(po,(r,a)=>{let n=Ls(r.configuration.accessToken);return n?LC(n,r,a.payload,e):r}).addCase(ir,(r,a)=>{if(r.configuration.accessToken!==a.payload.accessToken)return r;let{accessToken:n}=a.payload;if(!n)return r;let o=Ls(n);return o?[MC,DC,NC].reduce((i,s)=>s(o,i),r):r}).addCase(At,(r,a)=>{var s;let n=Ls(r.configuration.accessToken);if(!n)return r;let o=VC(n,r,a.payload.searchHub,e);return LC(n,o,(s=a.payload)==null?void 0:s.pipeline,e)}).addCase(Ca,(r,a)=>{let n=Ls(r.configuration.accessToken);return n?yE(n,r,a.payload.userDisplayName,e):r})});var BC={organizationId:O,accessToken:O,platformUrl:new w({required:!1,emptyAllowed:!1}),name:new w({required:!1,emptyAllowed:!1}),analytics:new q({options:{required:!1},values:{enabled:new K({required:!1}),originContext:new w({required:!1}),originLevel2:new w({required:!1}),originLevel3:new w({required:!1}),analyticsMode:new w({constrainTo:["legacy","next"],required:!1})}})};function jC(){return{organizationId:"searchuisamples",accessToken:"xx564559b1-0045-48e1-953c-3addd1ee4457",organizationEndpoints:gs("searchuisamples")}}var UC=new Y({...BC,search:new q({options:{required:!1},values:{pipeline:new w({required:!1,emptyAllowed:!0}),searchHub:de,locale:de,timezone:de,authenticationProviders:new X({required:!1,each:O})}})});function _C(){return{...jC(),search:{searchHub:"default"}}}var CE={debug:lo,pipeline:fo,searchHub:go,search:J};function xE(e){var n;let t=e.configuration.search,r=((n=e.configuration.organizationEndpoints)==null?void 0:n.search)||void 0;return{...t,apiBaseUrl:r}}function vE(e){let t=OC(e.loggerOptions);AE(e.configuration,t);let r=bE(e.configuration,t),a=FE(t),n={...qC(e.configuration,t),apiClient:r,streamingClient:a},o={...e,reducers:CE,crossReducer:QC(t)},i=EC(o,n),s=xE(e);return s&&i.dispatch(At(s)),{...i,get state(){return i.state},executeFirstSearch(c=yu()){if(Ms(i.state))return;let u=I({legacy:c,next:Gy()});i.dispatch(u)},executeFirstSearchAfterStandaloneSearchBoxRedirect(c){let{cause:u,metadata:l}=c;if(Ms(i.state))return;let d=l&&u==="omniboxFromLink",p=I({legacy:d?xu(l):Cu(),next:d?Wy(l):zy()});i.dispatch(p)}}}function AE(e,t){try{UC.validate(e)}catch(r){throw t.error(r,"Search engine configuration error"),r}}function bE(e,t){let{search:r}=e;return new Ss({logger:t,preprocessRequest:e.preprocessRequest||Za,postprocessSearchResponseMiddleware:(r==null?void 0:r.preprocessSearchResponseMiddleware)||jc,postprocessFacetSearchResponseMiddleware:(r==null?void 0:r.preprocessFacetSearchResponseMiddleware)||Uc,postprocessQuerySuggestResponseMiddleware:(r==null?void 0:r.preprocessQuerySuggestResponseMiddleware)||_c})}function FE(e){return new lf({logger:e})}function M(e){let t,r=new Map,a=()=>r.size===0,n=o=>{try{let i=JSON.stri
```

Comment:
The source mapping file is missing can you add the "headless.esm.js.map" file to this project and they will need to be included in the distribution files.

Reply:
No, this will actually be removed in a subsequent PR from Coveo.